src/modules/endpoint/DatabaseConnector.js

```diff
@@ -82,6 +83,18 @@ type ContentCityJsonType = {|
   tunewsEnabled: boolean
 |}
 
+type ContentPoiJsonType = {|
```

Comment:
A JsonType should not have non-json types:
* LocaltionModel
* Moment
* Map<string, string>